authorization/http_server_test.go

```diff
@@ -895,7 +875,7 @@ func jsonDiff(s1, s2 string) (diff string, err error) {
 
 var authorizationCmpOptions = cmp.Options{
 	cmpopts.EquateEmpty(),
-	cmpopts.IgnoreFields(influxdb.Authorization{}, "ID", "Token", "CreatedAt", "UpdatedAt"),
+	cmpopts.IgnoreFields(influxdb.Authorization{}, "ID", "Token", "HashedToken", "CreatedAt", "UpdatedAt"),
```

Comment:
Could it be a good idea to make 
```
"ID", "Token", "HashedToken", "CreatedAt", "UpdatedAt"
```
Enums or constants? That way its not "magic strings"